### PR TITLE
DOCS-6656-changed-chart-names

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/app-dashboards.adoc [hunk 1]
```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
 | *Performance* a|
 
 * Average response time inbound
```
Comment (pgaliana) [inline modules/ROOT/pages/app-dashboards.adoc]: Average response time - Inbound

File: modules/ROOT/pages/app-dashboards.adoc [hunk 2]
```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
...
-* Average Response Time Grouped by Endpoint Inbound
+* Average response time grouped by endpoint - Inbound
 * Mule messages
 * Response time graphs: 99, 90, 75, and 50 percentile inbound
 * Average response time outbound
```
Comment (pgaliana) [inline modules/ROOT/pages/app-dashboards.adoc]: Average response time - Outbound

File: modules/ROOT/pages/app-dashboards.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]:
```suggestion
* Average response time - Inbound
```

File: modules/ROOT/pages/app-dashboards.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]:
```suggestion
* Average response time  - Outbound
```

File: modules/ROOT/pages/app-dashboards.adoc [hunk 3]
```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
...
-* Average response time grouped by endpoint outbound
-* Response time graphs: 99, 90, 75, and 50 percentile outbound
+* Average response time grouped by endpoint - Outbound
+* Response time graphs: 99, 90, 75, and 50 percentile - Outbound
 //* Response Time graphs: 99, 90, 75, and 50 Percentile Inbound
 //* Average Response Time: Outbound, Grouped by Endpoint Outbound
```
Comment (kbostrom) [inline modules/ROOT/pages/app-dashboards.adoc]: What about the capital letters and lack of hyphens in these: Percentile Inbound, Outbound, Grouped by Endpoint Outbound?

File: modules/ROOT/pages/app-dashboards.adoc [hunk 4]
```diff
@@ -120,22 +122,24 @@ If you don't see the *Connectors* tab, verify that you have a Titanium subscript
...
-* All outbound grouped by response type
-* Total failed inbound grouped by endpoint
-* Total failed outbound grouped by endpoint
+* All inbound requests grouped by response type
+* All outbound requests grouped by response type
+* Failed inbound requests: Response time by endpoint
+* Total failed inbound requests grouped by endpoint
+* Failed outbound requests: Response time by HTTP endpoint (only HTTP endpoints are shown)
+* Total failed outbound requests grouped by HTTP endpoint (only HTTP endpoints are shown)
 | *JVM* a|
```
Comment (kbostrom) [inline modules/ROOT/pages/app-dashboards.adoc]: HIDDEN, UNEDITED

* Par new collection count, Par new collection time
* Classes: Loaded, Loaded total, Unloaded
* Heap: Committed, Used
* Thread Count: Number of simultaneous requests
* JVM Uptime: Uptime of the JVM
* Par Eden graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
* Par Survivor graphs (Used, Max, Init, and Committed): Usage metrics for the Par Eden Space pool
* Metaspace graphs (Used, Max, Init, Committed): Usage metrics for Java memory space
* Code Cache graphs (Used, Max, Init, Committed): Usage metrics for the Code Cache pool
* Compressed Class Space graphs (Used, Max, Init, Committed): Usage metrics for values of the metadata for compressed class pointers
* Committed Virtual Memory: Amount of virtual memory guaranteed to be available to the running process
| *Infrastructure* a|
* Application Process CPU Load: Amount of CPU load used by the JVM
* Application Process CPU Minutes: CPU time used by the process on which the JVM is running
* Operating System Swap Space: Amount of memory the system is swapping
* CPU Utilization %: Percentage of CPU used by each worker over time
* Memory Utilization: Amount of memory used by each worker over time
* Total System Processors: Number of system processors for workers available over time
* Total System Memory: Amount of system memory available for workers over time. +
If you have a Titanium subscription and the selected application is hybrid, this graph changes to *System Memory*.
* System Memory: This graph displays only if you have a Titanium subscription and the selected application is hybrid. This graph displays the total free system memory per worker.
* Thread Count - Server: Number of simultaneous requests for a worker over time
.Example: Infrastructure Dashboard
image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
| *Custom metrics* a|
The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
|===
== See Also
* xref:dashboard-custom-config.adoc[Configuring Custom Dashboards and Charts]
* xref:reports.adoc[Reports]
* xref:save-builtin-dashboard-as-custom.adoc[Save a Built-in Dashboard as a Custom Dashboard]

File: modules/ROOT/pages/app-dashboards.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]: These are commented out. I asked dev to let me know if they can be deleted. They're from before my time.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 5]
```diff
@@ -55,16 +55,17 @@ Built-in dashboards include various pages to view different types of charts:
 //* Memory: Number of mebibytes (MiB) used over time.
```
[the remark this answers is not in the document]
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]: Yes.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 6]
```diff
@@ -55,16 +55,17 @@ Built-in dashboards include various pages to view different types of charts:
...
 | *Inbound* a|
 
-* Total inbound requests
-* Average response time inbound
+* Total inbound requests by type
+* Average response time
```
[the remark this answers is not in the document]
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]: What do you mean?

File: modules/ROOT/pages/app-dashboards.adoc [hunk 7]
```diff
@@ -79,11 +80,12 @@ Inbound metrics are retained for 24 hours.
 //* Average response time inbound
 | *Outbound* a|
```
[the remark this answers is not in the document]
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]: Yes. These are names of charts that are available in the UI.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 8]
```diff
@@ -55,16 +55,17 @@ Built-in dashboards include various pages to view different types of charts:
...
 
-* Total inbound requests
-* Average response time inbound
+* Total inbound requests by type
+* Average response time
 * Average response time by endpoint
 ** Average: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
 ** Max: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 500ms; red if is greater than 500ms
 ** Min: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
+* Total requests by endpoint
```
[the remark this answers is not in the document]
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc]: They're documented in the order in which they appear in the UI.

**Pull request merged**